### ISSUE TITLE
[Docs] Discussion: either remove or curate the LearningResources site.

### ISSUE THREAD

**Issue by phryneas** [reduxjs/redux#4096]
Right now, [looking at the history](https://github.com/reduxjs/redux/commits/master/docs/introduction/LearningResources.md) the last "content update" to the ["Learning Resources](https://redux.js.org/introduction/learning-resources) page has been https://github.com/reduxjs/redux/pull/3232 in 2018.

That was pre-RTK - and yet, we link to these resources as a first entry point.

My suggestion would be to either 
* remove the page altogether
  Thich would be a shame for some timeless gems like 
  * Dan's initial Egghead course that does a very good job at explaining the contents - but as he states himself [show how it works, not how to use it](https://twitter.com/dan_abramov/status/1387400781507272704)
  * the "build your own redux" course
* remove most contents, annotate the rest as "old" or "just showing concepts" and ping the authors of what was there if they have newer courses including RTK that we can link.

**Comment by markerikson** [comment]
Yeah, the intent was to point to other resources outside our docs to provide some variety in explanations, but those really are outdated at this point.

I know that Jamund Ferguson recently did a "React-Redux hooks" course on Egghead ( https://app.egghead.io/playlists/modernizing-a-legacy-redux-application-with-react-hooks-c528 ) which ought to be useful.  Dave Ceddia's https://daveceddia.com/redux-tutorial/ is solid too.  Beyond that, I haven't been keeping tabs on meaningful resources for RTK + hooks.

**Comment by markerikson** [comment]
Did some updates in #4098 , but we can do more if there's other changes that need to be made.
